### PR TITLE
AI hud

### PR DESCRIPTION
**Pull request by Kelenius**
https://github.com/tgstation/-tg-station/pull/4048 with sprites from
https://github.com/tgstation/-tg-station/pull/4137

### CODE REVIEW

**Comment by Kelenius** [comment]
Aright, we good here. Removes a bunch of verbs and replaces then with hud buttons. Yey.

**Comment by PsiOmegaDelta** [comment]
Implying that removing verbs that be partially typed out and auto-completed quickly is a good thing.

**Comment by Kelenius** [comment]
But why would anyone do that when there are buttons?

**Comment by GinjaNinja32** [comment]
Because keyboard is faster than mouse when you know what you're doing. Pretty buttons are nice, and faster to begin with, but once you know what you're doing on keyboard it's so much more efficient.

**Comment by zaredman** [comment]
Hi I am someone who never ever uses buttons and always uses typed out verbs.

Don't remove verbs and think buttons are a 100% ok replacement.

**Comment by Kelenius** [comment]
Oh my god.

Fine.

I have an idea.

**Comment by Kelenius** [comment]
Is this acceptable to your tastes?

**Comment by zaredman** [comment]
Were *any* verbs removed?

**Comment by Kelenius** [comment]
"AI Core", because it's a duplicate of "Cancel Camera View".

File: code/modules/mob/living/silicon/ai/ai.dm [hunk 1]
```diff
@@ -3,28 +3,36 @@
...
-	/mob/living/silicon/ai/proc/core,
 	/mob/living/silicon/ai/proc/pick_icon,
 	/mob/living/silicon/ai/proc/sensor_mode,
 	/mob/living/silicon/ai/proc/show_laws_verb,
 	/mob/living/silicon/ai/proc/toggle_acceleration,
+	/mob/living/silicon/ai/proc/toggle_hologram_movement,
+	/mob/living/silicon/ai/proc/toggle_hidden_verbs,
+)
+
+var/list/ai_verbs_hidden = list( // For why this exists, refer to https://xkcd.com/1172/
```
Comment (GinjaNinja32) [inline code/modules/mob/living/silicon/ai/ai.dm]: I don't think you can *quite* make that comparison.

**Comment by sabiram** [comment]
anyone who links to xkcd should be summarily executed

**Comment by MrSnapwalk** [comment]
Oh hey, my old AI HUD sprites from back in the /NT/station days. Heh, didn't expect to see those again.

**Comment by Kelenius** [comment]
Any issues?

**Comment by Kelenius** [comment]
Aaaaany issues?

**Comment by Zuhayr** [comment]
As far as I know this was widely disliked on the staff chat due to the icons and the removal of verbs, I don't know if you're likely to get a solid answer one way or the other at the moment.

**Comment by Kelenius** [comment]
Yeah and I made the verbs toggleable.

**Comment by Kelenius** [comment]
My fucking god it's an interface improvement that doesn't remove anything now, what's so controversial about it

**Comment by Kelenius** [comment]
Also humans don't have verbs for dropping, swapping hands, toggling throw mode, changing run intent and many other UI functions and nobody seems to mind

**Comment by PsiOmegaDelta** [comment]
Many of those functions do have hotkeys instead.

**Comment by Spookerton** [comment]
We've had quite a lot of discussion on this pull request, and it's unfortunate that it's been open as long as it has because of that. Sorry.

For the most part we had issues with the removed verbs (fixed) and the button icons themselves (fixable), so this is :+1: from the general staff.

**Pull request merged**